v2/handler.go

```diff
@@ -80,6 +104,16 @@ type UnaryTransportHandler interface {
 	Handle(context.Context, *Request, *Buffer) (*Response, *Buffer, error)
 }
 
+// UnaryEncodingHandler handles a single, encoding-level, unary request.
```

Comment:
Perhaps explicate in a following sentence that an encoding handler handles a request after the request has been decoded into a concrete instance specific to the procedure.